marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/flow/testTargetCollectionAdditivity/suite-setup.xqy

```diff
@@ -0,0 +1,12 @@
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
```

Comment:
You can do this in setup.sjs, so you don't need this file (since you don't need to insert content below either, since flowRunner lets you pass in in-memory content).